civictechprojects/views.py

```diff
@@ -581,6 +581,44 @@ def recent_projects_list(request):
         return JsonResponse({'projects': hydrated_project_list})
 
 
+def limited_listings(request):
+    """Summarizes current positions in a format specified by the LinkedIn "Limited Listings" feature."""
+
+    def cdata(str):
+        # Using CDATA tags (and escaping the close sequence) protects us from XSS attacks when
+        # displaying user provided string values.
+        return f"<![CDATA[{str.replace(']]>', ']]]]><![CDATA[>')}]]>"
+    
+    def position_to_job(position):
+        project = position.position_project
+        roleTag = Tag.get_by_name(position.position_role.first().slug)
+
+        return f"""
+        <job>
+            <company>{cdata(project.project_name)}</company>
+            <title>{cdata(roleTag.display_name)}</title>
+            <description>{cdata(position.position_description)}</description>
+            <partnerJobId>{cdata(str(position.id))}</partnerJobId>
+            <location>{cdata(", ".join([project.project_city, project.project.state]))}</location>
```

Comment:
```suggestion
            <location>{cdata(", ".join([project.project_city, project.project_state]))}</location>
```

Reply:
Oh, great catch! I had fixed this locally (which is why the test data showed up correctly) but somehow it didn't make it into the github commit. I've addressed this and rebased onto origin/master.